install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librdb1`.
```

Comment:
@zeigerpuppy typo the package name is `librbd1`, please change the name in the rest of the document

Reply:
updated misspellings